tests/foreman/ui/test_contentview.py

```diff
@@ -716,11 +716,11 @@ def test_negative_add_dupe_repos(self):
                 common_locators['alert.success_sub_form']))
             with self.assertRaises(UIError) as context:
                 self.content_views.add_remove_repos(cv_name, [repo_name])
-                self.assertEqual(
-                    context.exception.message,
-                    'Could not find repo "{0}" to add into CV'
-                    .format(repo_name)
-                )
+            self.assertEqual(
+                context.exception.message,
+                'Could not find repo "{0}" to add into CV'
```

Comment:
prefix the formatting string with u name may contain unicode characters and that will fail on Python 2.